Formula/macvim.rb

```diff
@@ -14,20 +14,29 @@ class Macvim < Formula
   end
 
   option "with-override-system-vim", "Override system vim"
+  option "with-gettext", "Build MacVim with +gettext."
 
   deprecated_option "override-system-vim" => "with-override-system-vim"
 
+  conflicts_with "vim",
```

Comment:
It's been a while, but [last I checked](https://github.com/Homebrew/legacy-homebrew/pull/42314) we couldn't do conditional formulae conflicts.  Has that changed?